src/security/pages/sign-up.component.vue

```diff
@@ -0,0 +1,44 @@
+<script setup></script>
+
+<template>
+  <div class="flex flex-col items-center mt-8">
+    <div class="py-8">
+      <span class="text-2xl font-semibold"> Create an account </span>
```

Comment:
Don't leave whitespaces hanging around.

```suggestion
      <span class="text-2xl font-semibold">Create an account</span>
```